dt-assets/functions/cleanup.php

```diff
@@ -229,19 +224,16 @@ function dt_disable_wp_emoji() {
 }
 
 function dt_disable_emojis_tinymce( $plugins ) {
+    $plugins_array = [];
     if ( is_array( $plugins ) ) {
-        return array_diff( $plugins, array( 'wpemoji' ) );
-    } else {
-        return array();
+        $plugins_array = array_diff( $plugins, array( 'wpemoji' ) );
     }
+    return $plugins_array;
```

Comment:
@prykon this is simpler
```
    if ( is_array( $plugins ) ) {
        return array_diff( $plugins, array( 'wpemoji' ) );
    }
    return [];
```

Reply:
@prykon why keep the redundant variable?

Reply:
@corsacca Updated.